src/treeview/TableNode.ts

```diff
@@ -9,20 +9,23 @@ export class TableNode implements INode {
     public connectionName: string;
     public connectionLabel: string;
     public cache: TableDetails | undefined;
+    public source: 'tables' | 'favorites';
```

Comment:
If this is used multiple times, then add an interface. **Update:** this should be an enum. So `public source: TableSourceEnum` or something like that.